praw/models/user.py

```diff
@@ -106,7 +106,8 @@ def moderator_subreddits(self, **generator_kwargs):
         :class:`.ListingGenerator`.
 
         .. note:: This method will return a maximum of 100 moderated
-        subreddits, ordered by subscriber count.
+        subreddits, ordered by subscriber count. To retrieve more than
+        100 subreddits, use :meth:`.Redditor.moderated`.
```

Comment:
https://travis-ci.org/praw-dev/praw/jobs/618739213#L438

    docstring of praw.models.User.moderator_subreddits:7:Explicit markup ends without a blank line; unexpected unindent.

Therefore the docstring should be 

        .. note:: This method will return a maximum of 100 moderated
           subreddits, ordered by subscriber count. To retrieve more than
           100 subreddits, use :meth:`.Redditor.moderated`.

To check the output, run `sphinx-build -W docs <destdir>`.

Reply:
Thank you **so much**! I was so convinced that it was something wrong with the `code` section that I neglected to check the `note` part.